### ISSUE TITLE
WISE packaging in docker

### ISSUE THREAD

**Issue by advaitb** [slimgroup/WISE.jl#7]
Hi @ziyiyin97 ,

Have you tried packaging WISE scripts (or julia scripts) in docker. Whenever I do that, and instantiate the package it clones and downloads all packages from Manifest while running the script which makes the runtime longer. Is there a way to pre-download all libraries (like in python) and then just activate the package during runtime?

**Comment by ziyiyin97** [comment]
Good point! I am recently working with a cluster so a new run does not need to download all the Julia packages, but if you are working with cloud service then there can be a significant overhead for each run. I will find some time this week or next week to push out a docker image. Thanks for your feedback.
